config/jobs/kubernetes/kops/kops-config.yaml

```diff
@@ -178,26 +178,3 @@ presubmits:
         - --scenario=execute
         - --
         - ./hack/verify-packages.sh
-
-periodics:
-- interval: 30m
-  name: ci-kops-build
```

Comment:
need rebase again, I don't think you need to remove builds as part of this?

Reply:
The tests say that I have to remove it because apparently the sig-release board is the last place that job is run.  See discussion elsewhere on this PR which talks about this specific job.